Document/0x05d-Testing-Data-Storage.md

```diff
@@ -783,7 +783,16 @@ The [_Android Backup Extractor_](https://github.com/nelenkov/android-backup-extr
 ```bash
 java -jar android-backup-extractor-20160710-bin/abe.jar unpack backup.ab
```

Comment:
Can you remove the `android-backup-extractor-20160710-bin` part here? Because it would be nicer if you can just run it immediately.

Reply:
sure, no worries!